libs/product/src/drivers/magento/transforms/bundled-product-transformers.spec.ts

```diff
@@ -12,9 +13,13 @@ describe('DaffMagentoBundledProductTransformers', () => {
   });
 
 	describe('transform', () => {
+		const magentoBundledProduct = {
+			...magentoBundledProductData,
+			stock_status: MagentoProductStockStatusEnum.InStock
```

Comment:
I don't think these test changes are necessary.

Reply:
like that?

Reply:
whoa thats odd, it didn't fail on my machine

Reply:
my bad, I guess those test changes are necessary

Reply:
np, changed back